setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="dive_sailthru_client",
-    version="0.0.17",
+    version="0.0.18",
```

Comment:
if all goes well on merge this change will cause a new tag to be made in github, though in the past sometimes that has to be fixed manually. installations of this library conventionally depends on github tags, so this does need to be confirmed working before moving on to upgrading other installations (including divesite in the related PR for this, https://github.com/industrydive/divesite/pull/4430)